v2/driver/driver.go

```diff
@@ -73,7 +73,7 @@ func RootDAG(ctx context.Context, opts Options, mlmd *metadata.Client) (executio
 	if err != nil {
 		return nil, err
 	}
-	// TODO(Bobgy): change GCS output directory to pipeline root.
+	// TODO(v2): in pipeline spec, rename GCS output directory to pipeline root.
 	pipelineRoot := opts.RuntimeConfig.GetGcsOutputDirectory()
 	if pipelineRoot != "" {
```

Comment:
I feel confused about the logic here.
Looks like the the priority of pipeline root passed via arguments is higher than that of pipeline root defined in the configmap. It makes sense to me. But why the pipeline root passed via arguments will always be gcs?

Reply:
Yes, your understanding on priority is correct. Answered the gcs question in a common comment.